### ISSUE TITLE
Direwolf20 1.20 pack cannot be downloaded via the appimage or rpm

### ISSUE THREAD

**Issue by Drecondius** [FTBTeam/FTB-App#1205]
### Modpack

FTB Presents Direwolf20 1.20

### Modpack version

1.16.0

### Launcher

FTB App

### Has the pack been modified

No

### Log Files

this link is the console output that I could currently grab when running the app via terminal from /opt/FTB Electron App/ftb-app : https://hastebin.com/share/ucivaloqej.makefile

and this following link is similar console output to the previous but ran from terminal from the AppImage : https://hastebin.com/share/opoceguruj.makefile

### Describe the bug

simply put I try to download the direwolf20 1.20 modpack and it fails with an unknown error flashing across the bottom

### Steps to reproduce

run the appimage or the installed rpm, select the direwolf20 1.20 modpack, click install

### Expected behaviour

to download and install the modpack

### Additional information

If it's any consolation, the Direwolf20 1.20 modpack doesn't even showup on curseforge either.

**Comment by Drecondius** [comment]
Ok, will check it out.
Thanks for the suggestion.
